common/djangoapps/util/views.py

```diff
@@ -422,7 +445,14 @@ def build_error_response(status_code, field, err_msg):
             support_email=context["support_email"]
         )
 
-    _record_feedback_in_datadog(context["tags"])
+    # TODO: Determine if this info is actually used/needed.
```

Comment:
The code below should preserve the original behavior, but the question of whether or not this information is useful hasn't been answered.